### PR TITLE
add payments to dashboard

### PR DESCRIPTION
**Pull request by mini**
## Pull Request Information
**Relates to Github Issue:** #96 

**What is the PR Type:** Feature

**Describe the added/fixed behaviour:** Added payment component as shown in [HiFi](https://github.com/SE701ProjectGroup4/Flatmate-Management-System/wiki/High-Fidelity-Design#general)

capture 1
## Checklist

- [x] Pull request has suitable title, adequately described and associated with a github issue
- [x] Branch name follows correct convention (e.g. feature/[id]-some-feature)
- [x] Branch builds and executes with no errors
- [ ] Test cases has been generated for the feature and test cases pass
- [x] Updated relevant wiki/documentation

## Final Reminders
Please remember to squash the commits

### CODE REVIEW

File: Frontend/frontend/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "bootstrap": "^4.4.1",
```
Comment (HongShi10) [inline Frontend/frontend/package.json]: Are we using bootstrap or react bootstrap

Comment (mini) [reply Frontend/frontend/package.json]: react-bootstrap, but it uses bootstrap's css files

**Comment by mini** [comment]
Actually don't merge, this breaks the site as it includes the bootstrap CSS files. It is required but everyone else based their work on it not being present...

**Comment by mini** [comment]
Redoing this

**Pull request closed without merging**